app/endpoints.js

```diff
@@ -1,13 +1,27 @@
 import { buyingCatalogueApiHost, documentApiHost, identityServerUrl } from './config';
 
 const getSolutionListDataEndpoint = (apiHostUrl, filterType) => {
-  if (filterType === 'all') {
-    return `${apiHostUrl}/api/v1/Solutions`;
-  }
-  if (filterType === 'foundation') {
-    return `${apiHostUrl}/api/v1/Solutions/Foundation`;
+  const defaultUrl = `${apiHostUrl}/api/v1/Solutions`;
+  let url;
+
+  switch (filterType.toLowerCase()) {
+    case 'all':
+      url = defaultUrl;
+      break;
+
+    case 'foundation':
+      url = `${defaultUrl}/Foundation`;
+      break;
+
+    case 'nhsdgp001':
+    case 'dfocvc001':
+      url = `${defaultUrl}?frameworkId=${filterType}`;
+      break;
+
+    default: url = undefined;
   }
-  return undefined;
+
+  return url;
 };
```

Comment:
Rather than setting the value of `url` and returning it can we just return from the switch statement?

```javascript
switch (filterType.toLowerCase()) {
    case 'all':
      return defaultUrl;
...
```